### PR TITLE
Fixed - Google BigQuery date handling bug

### PR DESCRIPTION
**Pull request by guykeller**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/16925

## Description
Fixed the way Google BigQuery handles results from query in datetime format.
No they are returned properly and no error is returned.
Also added a task to the playbook that tests the command that used to return an error.

## Does it break backward compatibility?
   - No

## Must have
- [x] Tests
- [ ] Code Review

### CODE REVIEW

File: Integrations/GoogleBigQuery/GoogleBigQuery.py [hunk 1]
```diff
@@ -173,12 +190,12 @@ def test_module():
         query_job = bigquery_client.query(TEST_QUERY)
         query_results = query_job.result()
         results_rows_iterator = iter(query_results)
-        next(results_rows_iterator)
+        first_item = next(results_rows_iterator)
+        if str(first_item.get("name")) != "Ruby":
```
[the remark this answers is not in the document]
Comment (guykeller) [reply Integrations/GoogleBigQuery/GoogleBigQuery.py]: Added by mistake from a previous version, will remove

**Comment by guykeller** [comment]
As for unit tests - There really isn't anything to unit test, everything is included in the test playbook (including most of the possible arguments) and there isn't anything purely pythonic to test. What do you think?
As for the bug fixing process - will do

File: Integrations/GoogleBigQuery/GoogleBigQuery.py [hunk 2]
```diff
@@ -176,9 +184,7 @@ def test_module():
         next(results_rows_iterator)
         demisto.results("ok")
     except Exception as ex:
-        return_error("There was a problem creating a BigQuery client.\n"
```
Comment (Itay4) [inline Integrations/GoogleBigQuery/GoogleBigQuery.py]: please post screenshot of an error returned for example

File: Integrations/GoogleBigQuery/google_bigquery_test.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import datetime
+from GoogleBigQuery import convert_to_string_if_datetime
+
+
+def test_convert_to_string_if_datetime():
+    test1 = convert_to_string_if_datetime(None)
```
Comment (Itay4) [inline Integrations/GoogleBigQuery/google_bigquery_test.py]: name args in meaningful way
also add test for empty string

File: Integrations/GoogleBigQuery/GoogleBigQuery.py (hunk 2 above)
Comment (guykeller) [reply Integrations/GoogleBigQuery/GoogleBigQuery.py]: Added a more clear message

**Pull request merged**